ocs_ci/helpers/dr_helpers.py

```diff
@@ -612,6 +617,39 @@ def get_all_drpolicy():
     return drpolicy_list
 
 
+def get_managed_cluster_node_ips():
+    """
+    Gets node ips of individual managed clusters for enabling fencing on MDR DRCluster configuration
+    Returns:
```

Comment:
There should be one empty line before the returns section

Reply:
ack, made the change